modules/log.cpp

```diff
@@ -443,6 +444,12 @@ CModule::EModRet CLogMod::OnChanMsg(CNick& Nick, CChan& Channel, CString& sMessa
 	return CONTINUE;
 }
 
+CModule::EModRet CLogMod::OnWallops(const CNick& Nick, const CString& sMessage)
+{
+	PutLog("<" + Nick.GetNick() + "> " + sMessage, "wallops");
```

Comment:
How would you differentiate it from a conversation with someone with nick "wallops"?